### PR TITLE
docs: fee middleware tutorial improvements

### PR DESCRIPTION
**Pull request by srdtrk**
## Description

closes: #4854

### Commit Message / Changelog Entry

```text
docs: fee middleware tutorial improvements
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: docs/tutorials/01-fee/07-test-app.md [hunk 1]
```diff
@@ -105,15 +105,9 @@ hermes create channel --channel-version '{"fee_version":"ics29-1","app_version":
...
-
-```bash title="Terminal 4"
-hermes fee register-counterparty-payee --chain mars --channel channel-0 --port transfer --counterparty-payee cosmos1vapwvcsr0m32ptal6z6g9hjctywrw4yzyf6y6v
-```
 
-```bash title="Terminal 4"
-hermes fee register-counterparty-payee --chain earth --channel channel-0 --port transfer --counterparty-payee cosmos1uu38gkyed0dte5f9xk20p8wcppulsjt90s7f8h
-```
+Next recall that the Fee Middleware only pays fees on the source chain. That's why we should register `damian` and `charlie` as each other's counterparty on both chains.
+Luckily, the relayer does this for us under the hood because we've enabled the `auto_register_counterparty_payee` option in the config file.
```
Comment (crodriguezvega) [inline docs/tutorials/01-fee/07-test-app.md:110]: This `auto_register_counterparty_payee` option is only mentioned here. Should it be mentioned somewhere else as well when setting up hermes config file?

**Review by crodriguezvega — APPROVED** [review]
Thank you, @srdtrk.

File: docs/tutorials/01-fee/07-test-app.md (hunk 1 above)
Comment (srdtrk) [reply docs/tutorials/01-fee/07-test-app.md:110]: It is already added with [this](https://github.com/cosmos/ibc-go/pull/4979/files#diff-b84ca4e422c3888dcc4d8a5b55acdf7f96c340d26a522da1374d6f7447d0cce2L53). Docusaurus renders them as code snippets

**Review by damiannolan — APPROVED** [review]
🚀

File: docs/tutorials/01-fee/01-intro.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ This is a tutorial for wiring up the ICS-29 Fee Middleware to a Cosmos SDK block
 
 <HighlightBox type="prerequisite" title="Prerequisites">
 
-- Basic Knowledge of [Go](https://golang.org/doc/tutorial/getting-started)
-- Basic Knowledge of [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+- Basic Knowledge of [TypeScript](https://www.typescriptlang.org/)
 - Basic Knowledge of Cosmos SDK
     - If you are new to Cosmos SDK, we recommend you to go through the first two categories of the [Developer Portal](https://tutorials.cosmos.network/academy/1-what-is-cosmos/)
+- Basic Knowledge of [The Fee Middleware](https://ibc.cosmos.network/main/middleware/ics29-fee/overview)
```
Comment (charleenfei) [inline docs/tutorials/01-fee/01-intro.md:22]:
```suggestion
- Basic Knowledge of [the Fee Middleware module](https://ibc.cosmos.network/main/middleware/ics29-fee/overview)
```

**Review by charleenfei — APPROVED** [review]
nice cleanup @srdtrk 👍

File: docs/tutorials/01-fee/01-intro.md (hunk 2 above)
Comment (srdtrk) [reply docs/tutorials/01-fee/01-intro.md]: done

**Pull request merged**